srml/assets/README.adoc

```diff
@@ -0,0 +1,108 @@
+# Assets Module
+
+## Overview
+
+The assets module provides functionality for asset management of fungible asset classes with a fixed supply, including:
+
+* Asset Issuance
+* Asset Transfer
+* Asset Destruction
+
+The SRML Support Module implements the `Currency` trait abstraction over the SRML Assets Module's fungible assets system.
+
+### Types
+
+* `AssetId`: `u32`
+
+## Public Interface
+
+### Supported Origins
+
+**signed** - Used to issue, transfer, and destroy an asset holding.
+
+### Public Immutable functions (getters)
+
+#### `balance(id: AssetId, who: T::AccountId) -> T::Balance`
+
+#### `total_supply(id: AssetId) -> T::Balance`
+
+### Public Mutable functions (changing state)
```

Comment:
Public Dispatchable functions?
Public Mutable functions should be something other modules can call, similar to Public Immutable functions, which is not really the case here.